deployments/infrastructure.nix

```diff
@@ -62,6 +62,15 @@ with (import ./../lib.nix);
       };
       groups.production = {};
 
+      users.live-production = {
+        description     = "cardano live-production";
+        group           = "live-production";
+        createHome      = true;
+        isNormalUser = true;
+        openssh.authorizedKeys.keys = [domenKey georgeeeKey kosergeKey jakeKey];
```

Comment:
I see I'm authorized, but can't seem to log in yet. Should be `ssh live-production@<deployer ip>`, right?

Do we have to merge and deploy from master before it takes effect?